src/pages/patterns/filters.mdx

```diff
@@ -0,0 +1,383 @@
+
+### Advanced filtering
+
+Advanced Filters allow users to set and configure multiple filters on a dataset.
+
+#### Tooltip modal
+
+A Tooltip modal allows users to filter a dataset through one category with many options,
+and allows the user to search for specific options.
+They can also be part of a more complex flow.
+
+![Filter pattern advanced tooltip modal](figma:filter-usage-advanced-tooltip-modal.png)
+
+#### Sheet
+
+A Sheet is used when the dataset is filtered live as the user makes their selections.
+Selections are surfaced as a Tag as the user sets each filter.
+
+![Filter pattern advanced sheet 1](figma:filter-usage-advanced-sheet-1.png)
+![Filter pattern advanced sheet 2](figma:filter-usage-advanced-sheet-2.png)
+![Filter pattern advanced sheet 3](figma:filter-usage-advanced-sheet-3.png)
+![Filter pattern advanced sheet 4](figma:filter-usage-advanced-sheet-4.png)
+![Filter pattern advanced sheet 5](figma:filter-usage-advanced-sheet-5.png)
+![Filter pattern advanced sheet 6](figma:filter-usage-advanced-sheet-6.png)
+
+#### Drawer
+
+A Drawer is used to batch apply multiple filters at once. Selections are surfaced as a Tag
+after the user applies their filters and closes the Drawer.
+
+![Filter pattern advanced drawer 1](figma:filter-usage-advanced-drawer-1.png)
+![Filter pattern advanced drawer 2](figma:filter-usage-advanced-drawer-2.png)
+![Filter pattern advanced drawer 3](figma:filter-usage-advanced-drawer-3.png)
+
+## Behavior
+
+### Bulk actions
+
+When items are selected in a table and a filter is applied, the Filter overrides the selections and deselects them.
+
+![Filter pattern behavior bulk actions 1](figma:filter-behavior-bulk-actions-1.png)
+![Filter pattern behavior bulk actions 2](figma:filter-behavior-bulk-actions-2.png)
+
+## Localization & internationalization
+
+### RTL right-to-left
+
+![Filter pattern localization](figma:filter-localization.png)
+
+Filter entry points are mirrored for RTL languages.
+For more information about RTL considerations, see our [Buttons pattern](/patterns/buttons).
+
+## Accessibility
+
+### Tabbing order
+
+![Filter pattern accessibility tabbing order](figma:filter-accessibility-tabbing.png)
+
+1. User opens the filter container
+2. The focus moves to the first filter category and continues through all categories.
+3. If filters are applied live, focus moves to close filter. Filter can also be closed with the `esc` key.
+4. Focus moves to the first tag after container is closed.
+
+### Affordances
+
+Additional affordances should be included to ensure users are aware of the filters applied
+and the results of those filters. Screen readers should read out the results of a filter
+after the filtering actions are completed.
+
+### Additional resources
+
+[WAI-ARIA Combobox: "https://www.w3.org/WAI/ARIA/apg/patterns/combobox/"]
+[WAI-ARIA Listbox: "https://www.w3.org/WAI/ARIA/apg/patterns/listbox/"]
+[WAI-ARIA Menu: "https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/"]
```

Comment:
This would be good as a list I think. Also, fixed the link format. 😁 

```suggestion
- [WAI-ARIA Combobox](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/]
- [WAI-ARIA Listbox](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/]
- [WAI-ARIA Menu](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/]
```